src/components/PopupWithForm.js

```diff
@@ -4,6 +4,20 @@ export default class PopupWithForm extends Popup {
     super({ popupSelector });
     this._handleFormSubmit = handleFormSubmit;
     this._popupForm = this._popupElement.querySelector(".modal__form");
+    this._popupDeleteForm =
+      this._popupElement.querySelector("#delete-modal-form");
+    this._popupButton = this._popupElement.querySelector("#delete-sure-button");
+
+    this._pictureChangeButton = this._popupElement.querySelector(
+      "#profile-change-button"
+    );
+    this._pictureChangeModal = this._popupElement.querySelector(
+      "#picture-modal-form"
+    );
+    this._pictureChangeSubmit = this._popupElement.querySelector(
+      "#picture-save-button"
+    );
+    this._modalButton = this._popupElement.querySelector(".modal__button");
   }
 
```

Comment:
**Could be improved**

You can make ` setInputValues` in class `PopupWithForm`, which can insert data into inputs:

```js

  setInputValues(data) {
    this._inputList.forEach((input) => {
      // here you insert the `value` by the `name` of the input
      input.value = data[input.name];
    });
  }



```

So, this way you’ll not have to search the inputs of the profile in `index.js`